README.md

```diff
@@ -19,10 +19,16 @@ python3 -m pip install .
 ```
 
 ### Usage example:
+
 ```
 source <(ansibleinviewer -i inventory.yml)
 ```
 
+#### Possible flags
+
+* `-i`, `--inventory` - Path to ansible inventory
+* `-g`, `--groups` - Inventory groups of hosts to connect to
```

Comment:
I would like groups to option to be used like this: -g group1,group2,group3

Reply:
It's already possible. In new patchset I update it only to utilize : ans connector, because ! is bash restricted character